src/hotspot/cpu/riscv/riscv.ad

```diff
@@ -1185,14 +1185,13 @@ int MachCallRuntimeNode::ret_addr_offset() {
   //   la(t1, retaddr)
   //   la(t0, RuntimeAddress(addr))
   //   addi(sp, sp, -2 * wordSize)
-  //   sd(zr, Address(sp))
```

Comment:
`// for real runtime callouts it will be six instructions` should be five instructions now

Reply:
The 2 la macro instructions will be expand to 8 instructions. In XiaoLin's recent patch, he added detail in comments.
```
  //   la(t1, retaddr)                ->  auipc + addi
  //   la(t0, RuntimeAddress(addr))   ->  lui + addi + slli(C) + addi + slli(C) + addi
  //   addi(sp, sp, -2 * wordSize)    ->  addi(C)
  //   sd(zr, Address(sp))            ->  sd(C)
  //   sd(t1, Address(sp, wordSize))  ->  sd(C)
  //   jalr(t0)                       ->  jalr(C)
```
Check https://github.com/openjdk/riscv-port/pull/24/files#diff-e3c18b8b83898e82b5a3069319df6a47468e91cc2527bf065e704a685a20f26b

Reply:
Fixed

Reply:
Thanks. BTW, you can push another commit after starting the review instead of force pushing.

Reply:
Yes. To modify your patch in a pull request, just push more changes to the branch the pull request is based on.
Reference:
https://wiki.openjdk.java.net/display/SKARA/FAQ#FAQ-HowdoImodifythechangesinapullrequest?

Reply:
Thanks for the tip. New patch is pushed as a new commit.